language-support/ts/daml-json-types/BUILD.bazel

```diff
@@ -8,7 +8,7 @@ load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 ts_library(
     name = "daml-json-types",
     srcs = glob(["**/*.ts"]),
-    data = [":package.json"],
+    data = [":package.json", ":LICENSE"],
```

Comment:
Can't we use `//:LICENSE`?

Reply:
sadly we can't because the nodejs package rules wants all data files under the directory of the library.

Reply:
I understand. Thanks for the explanation.